### PR TITLE
init sigs list

### PR DESCRIPTION
**Pull request by rambohe-ch**

### CODE REVIEW

File: sig-list.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+As the number of projects in the OpenYurt community increases, it becomes more and more difficult
+to coordinate and synchronize state between projects.
+
+In order to solve the above issues, OpenYurt community discussed and prepared to create Special Interest Groups(SIGs)
+to organize and manage the projects. Each SIG with respect to a specific topic, such as ControlPlane or DataPlane.
```
Comment (gnunu) [inline sig-list.md:5]: Each SIG will be responsible for a ...?

File: sig-list.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+According to the current status of OpenYurt projects, initially we have launched 3 SIGs.
+- ControlPlane
+- DataPlane
+- IoT
+
+The projects included in each SIG are shown in the figure below:
+
+![openyurt-sigs](docs/img/openyurt-sigs.png)
+
+* SIG Scalability is still under discussion
```
Comment (gnunu) [inline sig-list.md:22]: discussion.
(missing peroid)

File: sig-list.md (hunk 1 above)
Comment (rambohe-ch) [reply sig-list.md:5]: fixed

File: sig-list.md (hunk 2 above)
Comment (rambohe-ch) [reply sig-list.md]: fixed

**Pull request merged**